src/main/frontend/src/routes/context.svelte

```diff
@@ -22,9 +27,16 @@
 </svelte:head>
 
 <!-- add newbie tooltips -->
-<CardForm title="Context" next="aggregates" bind:valid>
-	<TextField class="mb-4 pb-4" placeholder="com.example" bind:value={groupId} rules={[requireRule, packageRule]} validateOnBlur={!groupId}>Group Id</TextField>
-	<TextField class="mb-4 pb-4" placeholder="demo" bind:value={artifactId} rules={[requireRule, artifactRule]} validateOnBlur={!artifactId}>Artifact Id</TextField>
-	<TextField class="mb-4 pb-4" placeholder="1.0.0" bind:value={artifactVersion} rules={[requireRule, versionRule]} validateOnBlur={!artifactVersion}>Artifact Version</TextField>
-	<TextField class="mb-4 pb-4" placeholder="com.example.demo" bind:value={packageName} rules={[requireRule, packageRule]} validateOnBlur={!packageName}>Base Package Name</TextField>
+<CardForm title="Context" previous="platform" next="aggregates" bind:valid>
+	{#if $platformSettings && $platformSettings.platform === '.NET'}
+		<TextField class="mb-4 pb-4" bind:value={projectName} rules={[requireRule]}>Project Name</TextField>
+		<TextField class="mb-4 pb-4" bind:value={outputDirectory} rules={[requireRule]}>Output Directory</TextField>
+		<TextField class="mb-4 pb-4" bind:value={solutionFile} rules={[requireRule]}>Solution File</TextField>
+		<TextField class="mb-4 pb-4" bind:value={projectPath} rules={[requireRule]}>Project Path</TextField>
```

Comment:
Call it `Solution directory` which will be a path to the where the solution file named `Solution Name` will be stored

Reply:
After that please add a checkbox named 'Put solution and project in the same directory`. If not checked the following structure should be generated:

```
.../<SOLUTION_NAME>/<SOLUTION_NAME>.sln 
.../<SOLUTION_NAME>/<PROJECT_NAME>/<project files>
```

if checked the following structure will be generated

```
.../<SOLUTION_NAME>/<SOLUTION_NAME>.sln 
.../<SOLUTION_NAME>/<project files>
```

Then we could add a dropdown with the selection of available SDK

```
3.1.402
2.1.300
```

Then this is pretty obligatory, a dropdown (or textbox) with the `Framework` values:

```
netcoreapp2.2
netcoreapp3.0
netcoreapp3.1
net5.0*
...etc
```

The whole list is here https://docs.microsoft.com/fr-fr/dotnet/standard/frameworks (look TFM values) but it might be better to add a textbox and check if this value is contained in either TFM values, or allow the underlying generator blow an error. It's up to you